lib/fieldhand/get_record_parser.rb

```diff
@@ -0,0 +1,23 @@
+require 'fieldhand/record'
+
+module Fieldhand
+  # A parser for GetRecord responses
+  #
+  # See https://www.openarchives.org/OAI/openarchivesprotocol.html#GetRecord
+  class GetRecordParser
```

Comment:
Maybe these response-specific parsers are overkill as they only vary in two ways:

* The path of items sent to `Ox::Element#locate`
* The type of item to instantiate (e.g. `Record`, `Header`, etc.)

I could push these all back into a single `ResponseParser` which takes the above two arguments and then at least we will have separated parsing from pagination.